### ISSUE TITLE
Simple way to disable a set of cops (Style cops in my case)

### ISSUE THREAD

**Issue by lepieru** [bbatsov/rubocop#4463]
I want to add RuboCop's checks to a project but I don't want to be too rude with coding style. So I am looking for a simple way to disable all style cops for now.

It seems to be a classical case and it could be a nice way to incrementally adopt RuboCop.

--------

## Expected behavior

I would like to disable a set of cops, like Style cops in my case.

```yml
Style/*:
  Enabled: false
```

Could also be `StyleCops` instead of `Style/*`, it is just a proposition.

## Actual behavior

I have to manually insert all style cops in the `.rubocop.yml` file or I missed something.

## RuboCop version

```sh
$ rubocop -V
0.49.1 (using Parser 2.4.0.0, running on ruby 2.3.1 x86_64-linux-gnu)
```

**Comment by Drenmi** [comment]
You can disable a group like this:

```yml
Style:
  Enabled: false
```

Also, the fear of being rude is one of the selling points of programmatic enforcement. If the computer told you to do something, it doesn't affect your interpersonal relationships. 😉

**Comment by lepieru** [comment]
Oups, my bad.

> Also, the fear of being rude is one of the selling points of programmatic enforcement. If the computer told you to do something, it doesn't affect your interpersonal relationships. :wink:

I want RuboCop to be appreciate by new users, me included. And because many style cops are matters of taste, I prefer to disable them for now.

Thank you for this quick answer!

**Comment by Drenmi** [comment]
> I want RuboCop to be appreciate by new users, me included. And because many style cops are matters of taste, I prefer to disable them for now.

Understood. 😊  Most cops are configurable, so you can slowly adapt a few at a time. You can find the configuration options in [default.yml](https://github.com/bbatsov/rubocop/blob/master/config/default.yml), and generally the cop documentation will have information about the different options. 🙂
